.autover/changes/7403aeda-5b1c-4ce1-a88b-d15894f9e6d0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.TestTool",
+      "Type": "Minor",
+      "ChangelogMessages": [
+        "Add the ability to save requests and hide UI elements"
```

Comment:
can you update the readme with steps for how to do saved requests

Reply:
Done